Tests/conf.json

```diff
@@ -2111,6 +2109,8 @@
         "Tanium": "issue 15497",
         "Tenable.sc": "unstable instance",
         "Tenable.io": "Issue 16115",
+        "SlackV2": "Issue 19558",
```

Comment:
Is this part of mcaffe?

Reply:
no. I merged it from master and this was in there. It must have been removed in the process. I'll remove it.